xml/gl.xml

```diff
@@ -47743,7 +47743,7 @@ typedef unsigned int GLhandleARB;
             </require>
         </extension>
         <extension name="GL_NV_shader_thread_shuffle" supported="gl|glcore"/>
-        <extension name="GL_NV_shading_rate_image" supported="gl|glcore">
+        <extension name="GL_NV_shading_rate_image" supported="gl|glcore|gles2">
```

Comment:
Since there are parts of the extension API that are GL-only, you can mark up the XML to indicate this (and it will result in those APIs only appearing in glext.h, not gl2ext.h). For an example, see the XML for GL_EXT_blend_minmax - the \<require> block is interfaces appearing in all the APIs, while the \<require api="gl"> block is GL-specific interfaces.

Reply:
Thanks for looking at this. I think all functions and enums here can be supported by both GL and ES. Some others such as `glEnable(GL_MULTISAMPLE)`, `glGetDouble` and `glGet*Indexed` aren't in ES but also aren't introduced by the extension.